server/market/market.go

```diff
@@ -1420,6 +1415,8 @@ func (m *Market) unbookedOrder(lo *order.LimitOrder) {
 			epochIdx: -1, // NOTE: no epoch
 		},
 	})
+
+	// TODO: send revoke_order to owner
```

Comment:
@itswisdomagain Note that this is where I think we'd be sending the revoke_order notification.  Let me know before you dive into this because it might be best to build on this PR or for me to hit it.  Poking around now...

Reply:
No longer poling around since you are on it.  Go for it @itswisdomagain

Reply:
OK, scratch that.  It fits in with this PR so I'm gonna bang out the ntfn now